modules/local/fusioncatcher_downloadgenome/functions.nf

```diff
@@ -0,0 +1,68 @@
+//
+//  Utility functions used in nf-core DSL2 module files
+//
+
+//
+// Extract name of software tool from process name using $task.process
+//
```

Comment:
I would personally change the folder structure for the fusion tool modules like this
```
modules
  fusioncatcher
    download/
    detection/
  arriba
    download/
    detection
  ...
```

Reply:
Make sense. I did it in this way now. Looks clean.